ModAssistant/Localisation/sv.xaml

```diff
@@ -0,0 +1,232 @@
+
+    <!-- Mods Page -->
+    <sys:String x:Key="Mods:Title">Mods</sys:String>
+    <sys:String x:Key="Mods:Header:Name">Namn</sys:String>
+    <sys:String x:Key="Mods:Header:Installed">Installerad</sys:String>
+    <sys:String x:Key="Mods:Header:Latest">Senaste</sys:String>
+    <sys:String x:Key="Mods:Header:Description">Beskrivning</sys:String>
+    <sys:String x:Key="Mods:Header:Uninstall">Avinstallera</sys:String>
+    <sys:String x:Key="Mods:UninstallButton">Avinstallera</sys:String>
+    <sys:String x:Key="Mods:LoadFailed">Det gick inte att ladda modlistan</sys:String>
+    <sys:String x:Key="Mods:CheckingInstalledMods">Kollar efter installerade mods</sys:String>
+    <sys:String x:Key="Mods:LoadingMods">Laddar Mods</sys:String>
+    <sys:String x:Key="Mods:FinishedLoadingMods">Mods färdigladdade!</sys:String>
+    <sys:String x:Key="Mods:InstallingMod">Installerar {0}</sys:String>
+    <sys:String x:Key="Mods:InstalledMod">Installerat {0}</sys:String>
+    <sys:String x:Key="Mods:FinishedInstallingMods">Mods färdiginstallerade!</sys:String>
+    <sys:String x:Key="Mods:ModDownloadLinkMissing">Det gick inte att hitta någon nedladdningslänk för {0}</sys:String>
+    <sys:String x:Key="Mods:UninstallBox:Title">Avinstallera {0}?</sys:String>
+    <sys:String x:Key="Mods:UninstallBox:Body1">Är du säker på att du vill ta bort {0}?</sys:String>
+    <sys:String x:Key="Mods:UninstallBox:Body2">Detta kan leda till att dina andra mods slutar att fungera</sys:String>
+    <sys:String x:Key="Mods:FailedExtract">Extrahering misslyckades {0}, försöker igen om {1} sekunder. ({2}/{3})</sys:String>
+    <sys:String x:Key="Mods:FailedExtractMaxReached">Misslyckades med att extrahera {0} efter maxantalet försök ({1}), hoppar över. Denna mod kanske inte fungerar som den ska så fortsätt på egen risk.</sys:String>
+
+    <!-- About Page -->
+    <sys:String x:Key="About:Title">Om</sys:String>
+    <sys:String x:Key="About:PageTitle">Om Mod Assistant</sys:String>
+    <sys:String x:Key="About:List:Header">I'm Assistant, and I made Mod Assistant for mod assistance, med några grundläggande principer i åtanke:</sys:String>
+    <sys:String x:Key="About:List:Item1">Enkelhet</sys:String>
+    <sys:String x:Key="About:List:Item2">Portabilitet</sys:String>
+    <sys:String x:Key="About:List:Item3">En enkel, körbar fil</sys:String>
+    <sys:String x:Key="About:List:Item4">Ansvarsfullt användande</sys:String>
+    <Span x:Key="About:SupportAssistant">
+        Om du gillar programmet och vill stödja mig, besök gärna min
+        <Hyperlink local:HyperlinkExtensions.IsExternal="True" NavigateUri="https://bs.assistant.moe/Donate/">
+            donationssida
+        </Hyperlink>
+        eller min
+        <Hyperlink local:HyperlinkExtensions.IsExternal="True" NavigateUri="https://www.patreon.com/AssistantMoe">
+            Patreon
+        </Hyperlink>
+    </Span>
+    <sys:String x:Key="About:SpecialThanks">Special Thanks ♥</sys:String>
```

Comment:
This appears to be untranslated. Is this right?

Reply:
I'd say it looks better! "Special thanks" is an expression with too much cultural meaning put into it for a translation to make much sense. "Särskilt tack till" is the only possible translation I can think of, and it sounds pretty awkward.

Reply:
Ah, alright. No problem 👍